src/main/scala/io/scalac/tezos/translator/Boot.scala

```diff
@@ -28,7 +28,6 @@ object Boot {
     val host = httpConfig.getString("host")
     val port = httpConfig.getInt("port")
     val configuration = Configuration.getConfig(log)
-    log.info(s"Config loaded - $configuration")
```

Comment:
why you reomed this ?

Reply:
cos it contains f.ex. password for email user account we send msg from